tools/validators/instance_validator/validate/instance_parser.py

```diff
@@ -15,10 +15,17 @@
 """Parses and validates YAML instance files for syntax"""
 
 from __future__ import print_function
+
+import re
 import strictyaml as syaml
 import ruamel
 import sys
 
+_ENTITY_INSTANCE_REGEX = '[A-Z][A-Z0-9\\-]+:'
```

Comment:
this would be faster no and more less error prone if led by a ^.

'^[A-Z][A-Z0-9\\-]+:'

Reply:
done the extra `\\` is for pylint complaining